build/pkgs/free_fonts/SPKG.rst

```diff
@@ -0,0 +1,10 @@
+free fonts
```

Comment:
```suggestion
free_fonts: SOME DESCRIPTION HERE
```

Reply:
Done.

Reply:
Changed "free fonts: ..." to "free_fonts: ..."